deny.toml

```diff
@@ -0,0 +1,73 @@
+[advisories]
+db-path = "~/.cargo/advisory-db"
+db-urls = ["https://github.com/rustsec/advisory-db"]
+
+vulnerability = "deny"
```

Comment:
We should add a security threshold so this doesn't trigger on every little thing. Medium?

Threshold for security vulnerabilities, any vulnerability with a CVSS score lower than the range specified will be ignored. Note that ignored advisories will still output a note when they are encountered.
* None - CVSS Score 0.0
* Low - CVSS Score 0.1 - 3.9
* Medium - CVSS Score 4.0 - 6.9
* High - CVSS Score 7.0 - 8.9
* Critical - CVSS Score 9.0 - 10.0
`severity-threshold = `

Reply:
I'm not against this but I'd rather have the heads up so we can review it ourselves and then make a new commit explicitly allowing it. I will note the CI doesn't even run advisory checks right now for this reason, but I currently lean towards having it check everything and doing the aforementioned solution, having given it a second thought. Will reach out off GitHub to discuss.